frontend/encore/installation.rst

```diff
@@ -25,6 +25,14 @@ is also created/updated. When using npm 5, a ``package-lock.json`` file is creat
 
 Next, create your ``webpack.config.js`` in :doc:`/frontend/encore/simple-example`!
 
+.. tip::
+
+    If you are using Flex for your project, you can install Encore via:
+
+.. code-block:: terminal
+
+    $ composer require encore
```

Comment:
The code block needs to be indented by four spaces if it should be part of the tip.

Reply:
indeed, fixed